src/mmw/js/src/draw/controllers.js

```diff
@@ -51,6 +55,29 @@ var DrawController = {
     }
 };
 
+/**
+ * If we are in embed mode then the project is an activity and we want to keep
+ * the same project reguardless of changes to the AOI. This prepares a project
+ * immedialty upon visiting the page and will be the only project the user can
+ * save during this session.
+ */
+function enableSingleProjectModeIfActivity() {
+    if (settings.get('activityMode')) {
+        if (!App.currProject) {
+            var project = new modelingModels.ProjectModel({
+                name: 'New Project',
```

Comment:
Since we have the `is_activity` flag, it allows us (the backend people) to easily differentiate activities and projects, and it isn't different enough to enable users to clearly differentiate between the two. I like the "New Activity" idea, which is clear and explicit.

Any thoughts on this, @mmcfarland ?